apps/src/netsim/NetSimMessage.js

```diff
@@ -59,7 +62,8 @@ var NetSimMessage = module.exports = function (shard, messageRow) {
    * All other message content, including the 'packets' students will send.
    * @type {*}
    */
-  this.payload = messageRow.payload;
+  var b64 = messageRow.base64Payload;
+  this.payload = (b64) ? base64ToBinary(b64.string, b64.len) : undefined;
```

Comment:
We don't have a style guide rule about it, feel free to keep if you prefer them.